merlion/models/automl/autoets.py

```diff
@@ -31,9 +31,9 @@ class AutoETSConfig(SeasonalityConfig):
     def __init__(
         self, model: Union[ETS, dict] = None,
         auto_seasonality: bool = True,
-        auto_error: bool = True,
-        auto_trend: bool = True,
-        auto_seasonal: bool = True,
+        auto_error: bool = False,
+        auto_trend: bool = False,
+        auto_seasonal: bool = False,
         auto_damped: bool = False,
```

Comment:
I think it's fine to keep these as `True`, just update the test behavior accordingly.